oval_graph/oval_node.py

```diff
@@ -1,18 +1,18 @@
 '''
     Modules form my lib and for create ID
 '''
-import oval_graph.xml_parser
-import oval_graph.evaluate
 import uuid
 
+import oval_graph.evaluate
+
 
 '''
     This module contains methods and classes for
     constructing and controlling an oval tree.
 '''
 
 
-class OvalNode():
```

Comment:
Why that? The [standard recommends CapWords](https://www.python.org/dev/peps/pep-0008/#class-names)